Source/WebCore/platform/graphics/skia/ShareableBitmapSkia.cpp

```diff
@@ -91,6 +96,13 @@ PlatformImagePtr ShareableBitmap::createPlatformImage(BackingStoreCopy backingSt
     return SkImages::RasterFromData(m_configuration.imageInfo(), pixelData, bytesPerRow());
 }
 
+#if !USE(GTK4)
+RefPtr<cairo_surface_t> ShareableBitmap::createCairoSurface()
+{
+    return skiaImageToCairoSurface(*createPlatformImage().get());
+}
+#endif
```

Comment:
I prefer not to add cairo and GTK specific code to ShareableBitmapSkia, you can either call skiaImageToCairoSurface from the caller or create the cairo surface directly from the shareable bitmap data like we do in AcceleratedBackingStoreDMABuf:

```
m_surface = cairo_image_surface_create_for_data(static_cast<unsigned char*>(m_bitmap->data()), CAIRO_FORMAT_ARGB32, m_size.width(), m_size.height(), m_bitmap->bytesPerRow());
m_bitmap->ref();
static cairo_user_data_key_t s_surfaceDataKey;
cairo_surface_set_user_data(m_surface.get(), &s_surfaceDataKey, m_bitmap.get(), [](void* userData) {
    static_cast<WebCore::ShareableBitmap*>(userData)->deref();
});
```